internal/app/wwctl/container/imprt/root.go

```diff
@@ -39,7 +39,7 @@ Imported containers are used to create bootable VNFS images.`,
 func init() {
 	baseCmd.PersistentFlags().BoolVarP(&SetForce, "force", "f", false, "Force overwrite of an existing container")
 	baseCmd.PersistentFlags().BoolVarP(&SetUpdate, "update", "u", false, "Update and overwrite an existing container")
-	baseCmd.PersistentFlags().BoolVarP(&SetBuild, "build", "b", false, "Build container when after pulling")
+	baseCmd.PersistentFlags().BoolVarP(&SetBuild, "build", "b", true, "Build container when after pulling")
 	baseCmd.PersistentFlags().BoolVar(&SetDefault, "setdefault", false, "Set this container for the default profile")
```

Comment:
If you're going to remove SetDefault in `container build`, you should also remove it in `container import`.

It also needs to be removed from `ContainerImport` and `ContainerBuild`.

Reply:
Also please remove it from the documentation.